content/zh/_index.md

```diff
@@ -78,13 +77,13 @@ OpenTelemetry 是 100% 免费和开源的，
 
 {{% blocks/section color="secondary" type="cncf" %}}
 
-**OpenTelemetry 是一个 [CNCF][] [Incubating][] 项目**。<br> 这个项目是在合并
-OpenTracing 和 OpenCensus 项目之后成型的。
+**OpenTelemetry 是一个 [CNCF][] [孵化中][] 项目**。<br> 这个项目是由
```

Comment:
```suggestion
**OpenTelemetry 是一个 [CNCF][] [Incubating][] 项目**。<br> 这个项目是由
```
That's because [cncf site](https://www.cncf.io/projects/) does not has a zh version, both `incubating` and `孵化中` are acceptable IMO.

Reply:
CMIIW, we don't need to concern whether `zh` version CNCF site is available, since It's just a translation with a hyperlink that redirects users to the website we need.

Reply:
if translated, we may consider some options like:

- 孵化中项目
- 孵化中的项目
- 孵化级项目

Also, if the zh anchor does not work in hugo, it will be a little complicated.
We may need to remove the `[incubating]: https://www.cncf.io/projects/` from bottom and change the link to a markdown format like:

```
**OpenTelemetry 是一个 [CNCF][] [孵化级](https://www.cncf.io/projects/)项目**。<br> 这个项目是由
```

Reply:
I prefer the word "孵化级". 

```
**OpenTelemetry 是一个 [CNCF][] [孵化级][] 项目**。<br> 这个项目是由

...

[孵化级]: https://www.cncf.io/projects/
```

And those code changes should be fine. Could you raise some example that hugo goes wrong with `zh` anchor? I used it for many years and it works fine for me.

But since it's not refered many times, I think it's OK to use a inline link like:
```
**OpenTelemetry 是一个 [CNCF][] [孵化级](https://www.cncf.io/projects/)项目**。<br> 这个项目是由
```

Reply:
+1 孵化级 if translated.